content/docs/command-reference/plots/modify.md

```diff
@@ -26,6 +26,10 @@ The path to the metrics file `target` is required. It must be listed in a
 `dvc.yaml` file (see the `--plots` option of `dvc run`). `dvc plots modify` adds
 the display properties to `dvc.yaml`.
 
+Convert any
+[output or simple metrics file into a plot](#example-convert-any-output-into-a-plot)
+by passing it as a `target` to `dvc plot modify`.
+
```

Comment:
- Strange choice of which words are linked, please review.
- Let's be a bit more descriptive (less prescriptive) in cmd descriptions. Something like "Note that a secondary use of this command is to convert..."
- Should probably be the last paragraph in the desc.

Reply:
BTW does this also apply to `metrics modify`? Let's check, please. And if so include equivalent changes there.

Reply:
Updated the paragraph.
> apply to `metrics modify`

Metrics doesn't have `modify` subcommand

Reply:
Ah yeah 😅 OK then!